packages/newspringchurchapp/src/prayer/PrayerTab/index.js

```diff
@@ -0,0 +1,3 @@
+import PrayerTab from './PrayerTab';
```

Comment:
not sure why you need this file at all

Reply:
@cdanwards I totally agree with @redreceipt on this. I think you can move all this component stuff to the index and get rid of the `PrayerTab.js` file.